bumble/device.py

```diff
@@ -2808,11 +2808,23 @@ async def get_pin_code():
                 )
 
                 if pin_code is not None:
-                    pin_code = bytes(str(pin_code).zfill(6))
+                    # TODO: make pin code input more flexible,
```

Comment:
The `PairingDelegate` class should probably be enhanced with a `get_string` method (something like: `async def get_string(self, max_length)`

Reply:
Yes, that's what  I proposed in discussions with @uael. I will update it in another PR

Reply:
Would you be able to fix it on this one to avoid the TODO ? Thanks!

Reply:
Sure, I will

Reply:
Sg!